cmd/osbuild-worker/config_test.go

```diff
@@ -70,16 +70,14 @@ offline_token = "/etc/osbuild-worker/offline_token"
 type = "aws.ec2"
 iam_profile = "osbuild-worker"
 key_name = "osbuild-worker"
-cloudwatch_group = "osbuild-worker"
```

Comment:
Maybe leave it in the test with a comment `# deprecated in HMS-9304` to endure it does not break.